sdks/python/setup.py

```diff
@@ -225,48 +226,50 @@ def get_portability_package_data():
-        'dill>=0.3.1.1,<0.3.2',
-        # It is prudent to use the same version of pickler at job submission
-        # and at runtime, therefore bounds need to be tight.
-        # To avoid depending on an old dependency, update the minor version on
-        # every Beam release, see: https://github.com/apache/beam/issues/23119
-        'cloudpickle~=2.2.1',
-        'fastavro>=0.23.6,<2',
-        'fasteners>=0.3,<1.0',
-        'grpcio>=1.33.1,!=1.48.0,<2',
-        'hdfs>=2.1.0,<3.0.0',
-        'httplib2>=0.8,<0.23.0',
-        # numpy can have breaking changes in minor versions.
-        # Use a strict upper bound.
-        'numpy>=1.14.3,<1.25.0',   # Update build-requirements.txt as well.
-        'objsize>=0.6.1,<0.7.0',
-        'pymongo>=3.8.0,<5.0.0',
-        'proto-plus>=1.7.1,<2',
-        # use a tighter upper bound in protobuf dependency
-        # to make sure the minor version at job submission
-        # does not exceed the minor version at runtime.
-        # To avoid depending on an old dependency, update the minor version on
-        # every Beam release, see: https://github.com/apache/beam/issues/25590
-        'protobuf>=3.20.3,<4.24.0',
-        'pydot>=1.2.0,<2',
-        'python-dateutil>=2.8.0,<3',
-        'pytz>=2018.3',
-        'regex>=2020.6.8',
-        'requests>=2.24.0,<3.0.0',
-        'typing-extensions>=3.7.0',
-        'zstandard>=0.18.0,<1',
-      # Dynamic dependencies must be specified in a separate list, otherwise
-      # Dependabot won't be able to parse the main list. Any dynamic
-      # dependencies will not receive updates from Dependabot.
+      ],
+                            language_level=3),
+      install_requires=[
+          'crcmod>=1.7,<2.0',
+          'orjson<4.0',
+          # Dill doesn't have forwards-compatibility guarantees within minor
+          # version. Pickles created with a new version of dill may not unpickle
+          # using older version of dill. It is best to use the same version of
+          # dill on client and server, therefore list of allowed versions is
+          # very narrow. See: https://github.com/uqfoundation/dill/issues/341.
+          'dill>=0.3.1.1,<0.3.2',
+          # It is prudent to use the same version of pickler at job submission
+          # and at runtime, therefore bounds need to be tight.
+          # To avoid depending on an old dependency, update the minor version on
+          # every Beam release, see: https://github.com/apache/beam/issues/23119
+          'cloudpickle~=2.2.1',
+          'fastavro>=0.23.6,<2',
+          'fasteners>=0.3,<1.0',
+          'grpcio>=1.33.1,!=1.48.0,<2',
+          'hdfs>=2.1.0,<3.0.0',
+          'httplib2>=0.8,<0.23.0',
+          # numpy can have breaking changes in minor versions.
+          # Use a strict upper bound.
+          'numpy>=1.14.3,<1.25.0',  # Update build-requirements.txt as well.
+          'objsize>=0.6.1,<0.7.0',
+          'packaging>=22.0',
```

Comment:
This is the only change to this section, right? The diff here is rough

Reply:
Yeah the packaging dependency is the only new one, it looks like yapf saw fit to change some whitespace on every entry